.changeset/fluffy-horses-crash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@prefresh/vite': patch
```

Comment:
Removal of `Options` (if there isn't a better solution you know of, anyhow) might warrant a minor instead -- not sure how you want to handle it.